ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -206,6 +206,20 @@ def clip(
             subok=subok,
         )
 
+    def conjugate(
+        x,
+        /,
+        out=None,
+        *,
+        where=True,
+        casting="same_kind",
+        order="K",
+        dtype=None,
+        subok=True,
+    ):
+        return ivy.conj(x, out=out, where=where, casting=casting, order=order, dtype=dtype, subok=subok)
```

Comment:
The first argument to pass onto the `ivy` or `np_frontend` functions is supposed to be `self._ivy_array` not `x`.